account_analytic_required/README.rst

```diff
@@ -0,0 +1,100 @@
+=========================
+Account Analytic Required
+=========================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Faccount--analytic-lightgray.png?logo=github
+    :target: https://github.com/OCA/account-analytic/tree/16.0/account_analytic_required
+    :alt: OCA/account-analytic
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/account-analytic-16-0/account-analytic-16-0-account_analytic_required
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runboat-Try%20me-875A7B.png
+    :target: https://runboat.odoo-community.org/webui/builds.html?repo=OCA/account-analytic&target_branch=16.0
+    :alt: Try me on Runboat
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module adds an option *analytic policy* on accounts.
+You have the choice between 4 policies : *always*, *never*, *posted moves* and *optional*.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
```

Comment:
- Go to Accounting -> Configuration and activate: Analytic Accounting
- Go to... (I don't know hoy to continue) 😢

Reply:
I don't know why you don't see these lines:
```
Example:

If you want to have an analytic account on all your *expenses*,
set the policy to *always* for the account of type *expense*.
If you try to save a journal items with an account of type *expense*
without analytic account, you will get an error message.
```
anw, because "Account type" has been removed, the configuration will be done on the Account via menu: **Invoicing > Configuration > Accounting > Chart of Accounts**.